src/signals/incident-management/containers/IncidentOverviewPage/components/List/index.tsx

```diff
@@ -193,7 +197,11 @@ const List: FunctionComponent<ListProps> = ({
                     {getListIconByKey(priority, incident.priority?.priority)}
                   </StyledIcon>
                 </Td>
-                <Td detailLink={detailLink} data-testid="incidentId">
+                <Td
```

Comment:
It wouldn't allow for actually navigating to the incident, because there's no `<a />` on the `<tr />`. 

Although, I could add a key listener that navigates to the incident programatically. I'll take a look at that solution - visually it would be an improvement because the entire row is outlined.

Reply:
The `<tr />` is now focuseable through tabbing